main.go

```diff
@@ -237,8 +238,17 @@ func startProbe(probeConfig *probeConfig) {
 	timeout := time.After(probeTimeout)
 	receiveCount := 0
 
+	// Support for custom message payload
+	msgPayload := "This is msg %d!"
+	if probeConfig.MessagePayload != "" {
+		msgPayload = probeConfig.MessagePayload
+		// logger.Printf("Using custom message payload: %s", probeConfig.MessagePayload)
+	}
+
 	for i := 0; i < num; i++ {
-		text := fmt.Sprintf("this is msg #%d!", i)
+		// text := fmt.Sprintf("this is msg #%d!", i)
+		text := fmt.Sprintf(msgPayload, i)
+		logger.Printf("Msg payload: %s", text)
```

Comment:
This line might be good for debugging, however in a production setup this will spam the logs. I think we should remove this.

Reply:
Agreed, it adds to much verbosity.